apps/devhub/views.py

```diff
@@ -1754,6 +1754,16 @@ def render_agreement(request, template, next_step, step=None):
 
 
 @login_required
+@json_view
+def api_key_creds(request):
+    try:
+        credentials = APIKey.get_jwt_key(user=request.user)
+        return {'key': credentials.key, 'secret': credentials.secret}
```

Comment:
as mentioned above, there's no need to return the `key` (that doesn't need to be hidden). Also, I think it would be better if this endpoint accepted a `key` and returned the secret for that key, as long as the user owned those credentials. This would allow us to support [multiple credentials](https://bugzilla.mozilla.org/show_bug.cgi?id=1214928) in the future.